apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/dao/state/phasing/PhasingPollTable.java

```diff
@@ -149,7 +149,7 @@ public long getSenderPhasedTransactionFees(long accountId, int height) throws SQ
         }
     }
 
-    public DbIterator<Transaction> getHoldingPhasedTransactions(long holdingId, VoteWeighting.VotingModel votingModel,
+    public List<Transaction> getHoldingPhasedTransactions(long holdingId, VoteWeighting.VotingModel votingModel,
```

Comment:
Connection con = null;
can be moved into try-with-resource block
`try (Connection con = ..... ) {`

Reply:
Fixed